.devcontainer/scripts/container-creation.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+# GitHub doesn't pull in submodules by default when mounting
+# the repo directory on the container so we do this post-create
+git submodule update --init --recursive
```

Comment:
Not sure if there is a flag on the JSON config we can use to say that the repo needs to be cloned recursively but this does the trick for now, albeit it makes initialization a bit slow.

Reply:
Yep, the container uses a cloned workspace but that clone won't contain the submodules. From my experience, codespaces are created on a per branch basis so this should work with different target branches.

Reply:
I thought I saw a bug or thread where they were talking about changing the clone to pass the option to clone submodules, but maybe it hasn't happened yet.